.travis.yml

```diff
@@ -2,6 +2,9 @@ sudo: false
 language: python
 cache: pip
 
+dist:
+  - trusty
```

Comment:
Necessary because https://travis-ci.community/t/python-builds-started-failing-to-download-2-7-12/3516